test/README.md

```diff
@@ -0,0 +1,28 @@
+# TorchServe Regression Tests
+
+This folder contains nightly regression tests execututed against TorchServe master.These tests use [POSTMAN](https://www.postman.com/downloads/) for exercising all the Management & Inference APIs.
+
+### Latest Test Run Status
+
+![Build Status](https://codebuild.us-east-1.amazonaws.com/badges?uuid=eyJlbmNyeXB0ZWREYXRhIjoiS1QvY3lIUEdUb3hZVWNnbmJ2SEZCdExRNmNkNW9EVk1ZaFNldEk4Q0h3TU1qemwzQ29GNW0xMGFhZkxpOFpiMjUrZVVRVDUrSkh2ZDhBeFprdW5iNjRRPSIsIml2UGFyYW1ldGVyU3BlYyI6IjlvcjRqSTNMTmNhcExZbUwiLCJtYXRlcmlhbFNldFNlcmlhbCI6MX0%3D&branch=master)
+
+[Test Logs](https://torchserve-regression-test.s3.amazonaws.com/torch-serve-regression-test/tmp/test_exec.log)
```

Comment:
@dhanainme Unable to access the s3 bucket -- getting access denied error so could not very the tests

Reply:
CodeBuild seems to upload the log as private after every run. Just fixed the permissions. 
@chauhang , Do take a look & let me know if everything is fine.

Reply:
@chauhang Resolving this comment, please reopen it if you still have access issues.